source/testnet/net/guides/run-node-windows.rst

```diff
@@ -0,0 +1,214 @@
+.. include:: ../../variables.rst
+.. _run-node-windows:
+
+================================
+Run and manage a node on Windows
+================================
+
+.. contents::
+   :local:
+   :backlinks: none
+
+This guide describes how organizations can run and manage a node on the Concordium network from a Windows computer. You can also run a node using :ref:`Docker <run-a-node>` or :ref:`Ubuntu <run-node-ubuntu>`.
```

Comment:
Perhaps we should be clearer that the Dockerised setup is only for Linux distributions.